bool start
bool stop
---
bool state
